ct.yaml

```diff
@@ -0,0 +1,9 @@
+# See https://github.com/helm/chart-testing#configuration
```

Comment:
In terms of organization, I'd prefer if this file was defined under `chart/`. I suggest:
1. We rename the `chart/ci/` to `chart/.ci/` to convey the directory is not part of the chart and for internal use only
2. Move / rename `ci-values.yaml` to `chart/.ci/values.yaml`
3. Move `ct.yaml` to `chart/.ci/ct.yaml`

```
chart/.ci/
  ct.yaml
  values.yaml
```

Reply:
Unfortunately the helm chart-testing utility is [hard-coded](https://github.com/helm/chart-testing/blob/0a093e464b884565148eb71dda6999de0c956b17/pkg/chart/chart.go#L225) to only recognize the folder named "ci" within a given chart. However, I might be able to configure the GitHub Actions workflow in a way that will allow for the ct.yaml file to be moved into the "ci" folder.